pullandrun.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+#
+# This needs valid creds so it can pull from git
+# prob should be a dedicated user that only has access to what it needs
+uname=<username>
```

Comment:
Can we move these credentials to a separate config file? The github user credentials will probably have to be read-write, and it'd be nice to keep those in a file that we can add to the .gitignore, so they don't end up in our repo.

Reply:
I can also make some credentials and send those over! I'll do that now.